src/pj_param.c

```diff
@@ -1,20 +1,78 @@
 /* put parameters in linked list and retrieve */
-#include <projects.h>
 #include <stdio.h>
+#include <ctype.h>
```

Comment:
Unless there is a specific reason, please alphabetize the includes.  If there is a reason for the order, please add a comment.

Reply:
Another case of classic 1980s proj code, where probably no harm is done by reordering.